### ISSUE TITLE
My VA: Sprint 30 (2/14/24 - 2/27/24) 

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#73609]
## Priorities
_At the end of this sprint, what do we want to have accomplished/delivered? What would we like to be able to present as a team in our demo?_
1. Get dot indicator et al. study positioned for kick-off (stretch goal start the sessions)
2. Continued testing and BE telemetry enhancements
3. Kick off onboarding initiative 

### Team News, Announcements & Availability
- President's Day 2/19
- AJ & Alex OOO 2/23
- Allison & Samara OOO 2/16

# Notifications

## Dot Indicator (#63788) 
**Research & Design**
- [x] #76141
- [x] #76283 
- [x] #76284 
- [ ] #75064 

**FE**
- [ ] #76142

## Onsite Notifications (#50758)
**FE**
- [ ] #74401 

# My VA

## My VA Claims and Appeals Improvements (#71287, #54403)
**PM, Research & Design**
- [ ] #71288 

## My VA Telemetry, Monitoring & Alerting & Testing Improvements (#71927, #61602)
**FE**
- [ ] #72516 
- [ ] #57462 

**BE**
- [ ] #71930 
- [ ] ~#66448~ _moved to next sprint_ 
- [ ] #76289 

## My VA Sketch > Figma Migration (#72326)
**Research & Design**
- [x] #76328 

## My VA/General Improvements (#31611)
**Research & Design**
- [x] #75057 

**FE**
- [ ] #65042 

# Identity & Personalization
## VA.gov Onboarding (#74163 )
**PM** 
- [x] #75799 

**Research & Design**
- [ ] #75797 
- [ ] #75880 

# Waiting on Deployment and/or Validation Only
- [x] #72514 
- [x] #72511 
- [x] #73088 

# PM Tasks (Varying initiatives)
- [ ] #70284
